grammarOperations.js

```diff
@@ -191,7 +190,11 @@ sem.addOperation('composeJson', {
   },
 
   sectionElementWithTitle: function (_, titleText) {
-    return titleText.sourceString
+    let textObj = titleText.composeJson()
+    if (textObj.length === 1){
```

Comment:
Could `textObj` ever by `None` or `undefined`?

Reply:
No. the rule behind it is
 `sectionElementWithTitle = sectionMarker (chapterContentTextContent | notesElement | milestoneElement)+ 
`

None of these can be empty. So, there should be something in titleText for it to be accepted by grammar.
Dedending on what is in the titleText, the `textObj` will have some content. It will not be None or undefined.